functests/sctp/sctp.go

```diff
@@ -483,3 +494,67 @@ func setupEgress(namespace, fromPod, toPod string, port int32) error {
 	_, err := client.Client.NetworkPolicies(namespace).Create(policy)
 	return err
 }
+
+func findSCTPNodeSelector() (string, error) {
+	mcList, err := client.Client.MachineConfigs().List(metav1.ListOptions{})
+	Expect(err).ToNot(HaveOccurred())
+
+	for _, mc := range mcList.Items {
+		if enablesSCTP(mc) {
+			newSCTPNodeSelector, err := findSCTPNodeSelectorByMCLabel(mc.ObjectMeta.Labels["machineconfiguration.openshift.io/role"])
```

Comment:
Not sure if the role is mandatory, but if a mc does not have a label this is going to panic.
Can you check the existence and continue if it's not there?